joomla/joomla-cms
# [4.0] Rename Purge to Clear Log

Pull Request for Issue #29947

### Summary of Changes
Rename Purge to Clear Log in toolbar for consistency with other Clear buttons.


### Testing Instructions
Code review.
or
Go to Users -> User Actions Log
See the Purge button in the Toolbar.
